package-overrides/npm/leaflet.markercluster@0.4.0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "shim": {
+    "dist/leaflet.markercluster": [
+      "leaflet"
+    ]
```

Comment:
Again, worth considering npm v github here. If it's working fine, and there aren't unnecessary installs happening, it's certainly fine to check this in though, but just let me know.